traptor/traptor.py

```diff
@@ -935,9 +955,10 @@ def run(self):
             self.twitter_rules = self._make_twitter_rules(self.redis_rules)
             self.logger.debug("Twitter rules: {}".format(self.twitter_rules.encode('utf-8')))
 
-            # Make the rule counters
+            # Make the rule and limit message counters
             if self.traptor_type != 'locations':
                 self._make_rule_counters()
+                self._make_limit_message_counter()
```

Comment:
Does Twitter API guarantee we'll never see a limit type message for location traptors? If not, then we have possible `NoneType` conditions and we should either alway init the counters, or do existence checks before we try to use them.